docs/_packages/web_api.md

```diff
@@ -246,7 +246,7 @@ pattern for people that use _functional programming_.
 ---
 
 ### Opening modals
-[Modals](https://api.slack.com/block-kit/surfaces/modals) can be created by calling the `views.open` method. The method requires you to pass a valid [view payload](https://api.slack.com/reference/block-kit/views) in addition to a `trigger_id`, which can be obtained when a user invokes your app using a slash command, clicking a button, or using [another interactive action](https://api.slack.com/reference/messaging/interactive-components).
```

Comment:
Would love any feedback here on terminology for 1.0 vs 2.0 - went with "with CLI" and "not CLI" but let me know if there's a better way to phrase this! 😬